apps/meteor/app/livechat/server/api/v1/room.ts

```diff
@@ -448,7 +448,20 @@ API.v1.addRoute(
 				delete guestData.phone;
 			}
 
-			await Promise.allSettled([Livechat.saveGuest(guestData, this.userId), Livechat.saveRoomInfo(roomData)]);
+			const pms = (await Promise.allSettled([Livechat.saveGuest(guestData, this.userId), Livechat.saveRoomInfo(roomData)])) as {
```

Comment:
Also, the allSettled call is unnecessary here. It's a rather small (just two) number of promises you're handling. On top of which manually checking for failure. Whereas just doing await sequentially the wrapper will take care of fail response in case either of the promise reject.